pkg/user/user.go

```diff
@@ -0,0 +1,44 @@
+package user
+
+import (
+	"os"
+	"os/user"
+	"runtime"
+	"strconv"
+)
+
+// Current - workaround for docker images with fully static binary and 32bit linux operating systems.
```

Comment:
I would word it as, a portable implementation to determine the current user. Golang's user.Current does not work reliably under docker or 32bit linux...